### ISSUE TITLE
support gRPC services

### ISSUE THREAD

**Issue by tmc** [tiziano88/elm-protobuf#2]
this could generate typed client code for grpc services

**Comment by tiziano88** [comment]
Indeed, it would be nice to generate gRPC stubs; I don't expect to be able to implement this in the short term, but it's definitely on my list. Feel free to send contributions if you wish

**Comment by tiziano88** [comment]
I think in order to support the _real_ gRPC protocol, we would have to first implement protobuf binary serialization (not just JSON) over HTTP2, which is a considerable amount of work. But I'm leaving this open in case anyone has any ideas on how to go about that!

**Comment by krestenkrab** [comment]
https://spatialos.improbable.io/games/grpc-web-moving-past-restjson-towards-type-safe-web-apis

**Comment by azr** [comment]
Would it be an option to generate the proto/grcp library in javascript using existing tools and then call it from elm ?

**Comment by awalterschulze** [comment]
Just a note:
grpc is supposed to be serialization agnostic, with a preference for protobufs.
Also since proto3, json is an official serialization format supported by the proto spec.

**Comment by qwfy** [comment]
Now we have [grpc-web](https://github.com/grpc/grpc-web), will this help?
